docs/sphinx/user_guide/feature/view.rst

```diff
@@ -82,6 +82,44 @@ accesses array entries with unit stride. The loop::
 
 access array entries with stride N :subscript:`n` * N :subscript:`(n-1)` * ... * N :subscript:`(j+1)`.
 
+MultiView
+^^^^^^^^^^^^^^^^
+
+A ``RAJA::MultiView`` object wraps an array-of-pointers,
+or a pointer-to-pointers, rather than a ``RAJA::View`` which wraps a single
+pointer or array. This allows a single ``RAJA::Layout`` to be applied to a
+large set of arrays, and encapsulates a set of arrays more easily.
+
+The instantiation of a MultiView works exactly like a standard View,
+except that it wraps an array-of-pointers::
+
+  int a1[4] = {5,6,7,8}; int a2[4] = {9,10,11,12}; // individual arrays of the same size
+  int * myarr[2]; // array of arrays which will be passed into MultiView
+  myarr[0] = a1; myarr[1] = a2;
+  RAJA::MultiView< int, RAJA::Layout<2> > MView(myarr, 2, 2);
+
+The MultiView can access individual arrays via the zero-th index::
+
+  MView( 0, 2, 2 ); // accesses 8, the (2,2)th element of the first array
+  MView( 1, 2, 2 ); // accesses 12, the (2,2)th element of the second array
+
+The index into the array-of-pointers can be moved (swizzled) to different
+indices of the ``()`` operator. This is achieved by instantiating the MultiView

```

Comment:
I would change how this is worded. It's also harder to understand if part of the explanation is in the test and part is in the code comment.  In general, use comments in the code examples sparingly to explain things. Specifically, change "This is achieved but instantiation...." to "By passing a third template  parameter to the MultiView constructor, the array entry index and the integer indicating which array to access can be reversed::"